src/java/org/apache/cassandra/triggers/TriggerExecutor.java

```diff
@@ -19,11 +19,6 @@
 package org.apache.cassandra.triggers;
 
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
```

Comment:
They shouldn't both be there, but this doesn't match C* code style, with the correct IntelliJ settings it should automatically be converting the * import into individual imports. You are going to constantly churn imports with everyone else if your IDE is set up incorrectly.

Reply:
This was just an automated removed unused, I think? Since unused imports break the build. The star import was already there. Happy to swap this to removing the star import if that is indeed below the threshold (might include additional classes of course)

Reply:
You are right the imports are redundant with the * import, but if you have the correct style settings then it would remove the star import not the individual imports.

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/25d50349-ba50-4ee1-8cca-825c3957a3d9">